plone/schemaeditor/tests/browser_testing.zcml

```diff
@@ -1,6 +1,12 @@
 <configure
     xmlns="http://namespaces.zope.org/zope"
-    xmlns:browser="http://namespaces.zope.org/browser">
+    xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:zcml="http://namespaces.zope.org/zcml">
+
+    <!-- only load PageTemplates zcml in Zope 4 -->
+    <configure zcml:condition="installed Products.PageTemplates.engine">
```

Comment:
No -- earlier versions of Zope also have zcml in Products.PageTemplates, but loading it here fails because it uses directives that aren't registered in this test setup. On Zope 4, which has the `engine` module in Products.PageTemplates, the zcml is simpler and works (and is necessary to render using Chameleon)